core/src/main/java/org/mule/runtime/core/internal/event/DefaultEventContext.java

```diff
@@ -185,7 +183,7 @@ public String toString() {
 
     private ChildEventContext(BaseEventContext parent, ComponentLocation componentLocation,
                               FlowExceptionHandler messagingExceptionHandler) {
-      super(messagingExceptionHandler, Mono.empty());
+      super(messagingExceptionHandler, null);
```

Comment:
same as in https://github.com/mulesoft/mule/pull/6147/files#diff-c7ba2cf2082917d74cf6465bc46d16bbR66